src/screens/AddSiteScreen.js

```diff
@@ -1,12 +1,25 @@
 import React, {Component} from 'react';
 import {Text} from 'react-native';
+import {Icon} from 'react-native-elements';
 
 import {campsite} from '../locale.en';
-import {backgroundLogo} from '../styles';
+import {backgroundLogo} from '../styles/index';
 
-import {CardSection, Card, Input, Button} from './common';
+import {CardSection, Card, Input, Button} from '../components/common/index';
+
+class AddSiteScreen extends Component {
+    static navigationOptions = (props) => {
+        const {navigation: {navigate}} = props;
+
+        return {
+            title: 'Add a Site',
+            headerTitle: 'Add a Campsite',
+            headerLeft: null,
+            tabBarIcon: ({focused, tintColor }) => (<Icon type='material-community' name={focused ? 'tent' : 'tent'} size={25} color={tintColor} />)
```

Comment:
Really great use of `react-navigation` together with `react-native-elements`. Things like `focused` and `tintColor` come for free, and so we can easily do things like toggle color or icon based on state.